### ISSUE TITLE
'transform' should be listed after '-webkit-transform', '-ms-transform'...

### ISSUE THREAD

**Issue by nitatemic** [nitatemic/AlexandreDeLemeny-Makedone_4_24052021#5]
When multiple versions of the same CSS property are specified, the last supported one will be used due to how browsers handle fallback values. This means the order matters when using both vendor-prefixed and unprefixed versions of the same property. Specifically, the unprefixed version must be listed last to ensure standardized behavior takes precedence.

https://webhint.io/docs/user-guide/hints/hint-css-prefix-order/?source=devtools

**Comment by nitatemic** [comment]
Done but I would have to add more compatibility property
